server/endpoint/health/searcher/endpoint_test.go

```diff
@@ -33,7 +33,7 @@ func Test_Health_Endpoint(t *testing.T) {
16"}},"id":"cxx2e","masters":[{"id":"hhn5a"}],"scaling":{"max":3,"min":3},"version":"","workers":[{"id":"hi5ie"},{"id":"pq295"},{"id":"oh8xv"}]},"kvm":{"masters":[{"cpus":2,"disk":40,"dockerVolumeSizeGB":0,"memory":"8G"}],"network":{"flannel":{"vni":4}},"workers":[{"cpus":2,"disk":40,"dockerVolumeSizeGB":0,"memory":"3G"},{"cpus":2,"disk":40,"dockerVolumeSizeGB":0,"memory":"3G"},{"cpus":2,"disk":40,"dockerVolumeSizeGB":0,"memory":"3G"}]},"versionBundle":{"version":"3.8.0"}},"status":{"cluster":{"conditions":[{"lastTransitionTime":"2019-09-23T12:08:24.66025979Z","status":"True","type":"Updating"},{"lastTransitionTime":"2019-09-23T12:01:19.857790161Z","status":"True","type":"Created"}],"network":{"cidr":""},"nodes":[{"labels":{"beta.kubernetes.io/arch":"amd64","beta.kubernetes.io/os":"linux","giantswarm.io/provider":"kvm","ip":"172.23.0.206","kubernetes.io/arch":"amd64","kubernetes.io/hostname":"master-hhn5a-66b4789bbc-tjvb4","kubernetes.io/os":"linux","kubernetes.io/role":"master","kvm-operator.giantswarm.io/version":"3.8.0","node-role.kubernetes.io/master":"","node.kubernetes.io/master":"","role":"master"},"lastTransitionTime":"2019-09-23T17:55:06.434468696Z","name":"master-hhn5a-66b4789bbc-tjvb4","version":"3.8.0"},{"labels":{"beta.kubernetes.io/arch":"amd64","beta.kubernetes.io/os":"linux","giantswarm.io/provider":"kvm","ip":"172.23.0.154","kubernetes.io/arch":"amd64","kubernetes.io/hostname":"worker-hi5ie-5f79675f59-5wgss","kubernetes.io/os":"linux","kubernetes.io/role":"worker","kvm-operator.giantswarm.io/version":"3.8.0","node-role.kubernetes.io/worker":"","node.kubernetes.io/worker":"","role":"worker"},"lastTransitionTime":"2019-09-23T17:55:06.434469572Z","name":"worker-hi5ie-5f79675f59-5wgss","version":"3.8.0"},{"labels":{"beta.kubernetes.io/arch":"amd64","beta.kubernetes.io/os":"linux","giantswarm.io/provider":"kvm","ip":"172.23.0.242","kubernetes.io/arch":"amd64","kubernetes.io/hostname":"worker-oh8xv-c549d84f-24z2s","kubernetes.io/os":"linux","kubernetes.io/role":"worker","kvm-operator.giantswarm.io/version":"3.5.0","node-role.kubernetes.io/worker":"","node.kubernetes.io/worker":"","role":"worker"},"lastTransitionTime":"2019-09-23T17:55:06.434470124Z","name":"worker-oh8xv-c549d84f-24z2s","version":"3.5.0"},{"labels":{"beta.kubernetes.io/arch":"amd64","beta.kubernetes.io/os":"linux","giantswarm.io/provider":"kvm","ip":"172.23.0.86","kubernetes.io/arch":"amd64","kubernetes.io/hostname":"worker-pq295-6c4bbcccf7-kql2k","kubernetes.io/os":"linux","kubernetes.io/role":"worker","kvm-operator.giantswarm.io/version":"3.5.0","node-role.kubernetes.io/worker":"","node.kubernetes.io/worker":"","role":"worker"},"lastTransitionTime":"2019-09-23T17:55:06.434470788Z","name":"worker-pq295-6c4bbcccf7-kql2k","version":"3.5.0"}],"resources":null,"scaling":{"desiredCapacity":0},"versions":[{"date":"0001-01-01T00:00:00Z","lastTransitionTime":"2019-09-23T12:01:19.910404547Z","semver":"3.5.0"}]},"kvm":{"nodeIndexes":{"hhn5a":1,"hi5ie":2,"oh8xv":4,"pq295":3}}}}`,
```

Comment:
Let's discuss the general idea at another point but the formatting should change for sure.
We sometimes put these things into extra files as a const - I am not sure about the usefulness of such a specific object inside the test.

Changes to the spec of the kvmConfig could have unforeseen / unfortunate consequences here. I am happy to be wrong though.

Reply:
On the one hand, I agree that this could be fragile. On the other hand, the whole health determination is based on this structure. If we change `KVMConfig`, we'll have to update this service too so I think they're necessarily tightly coupled. Open to suggestions for a better/more maintainable place to keep this data though.

Reply:
I reverted this. We'll figure it out once we're implementing health check logic.